brmp/fit.py

```diff
@@ -26,6 +28,166 @@
 
 
 class Fit(namedtuple('Fit', 'formula metadata contrasts data model_desc model samples backend')):
+
+    # TODO: This doesn't match the brms interface, but the deviation
+    # aren't improvements either. Figure out what to do about that.
+
+    # brms                                               | brmp
+    # -----------------------------------------------------------------------------------
+    # fitted(fit, summary=FALSE)                         | fitted(fit)
+    # fitted(dpar='mu', scale='linear', summary=FALSE)   | fitted(fit, 'linear')
+    # fitted(dpar='mu', scale='response', summary=FALSE) | fitted(fit, 'response')
+    # fitted(fit, newdata=..., summary=FALSE)            | fitted(fit, data=...)
+    # fitted(fit, ..., summary=TRUE)                     | summary(fitted(fit, ...))
+    # predict(fit, summary=FALSE)                        | fitted(fit, 'sample')
+    # predict(fit, summary=TRUE)                         | summary(fitted(fit, 'sample'))
```

Comment:
Does this comment need revising, or do we still have a predict function?

Reply:
The `predict` mentioned here is in brms. I'll update the brmp column to reflect this PR. I expect this comment to disappear once we decide about #32.

Reply:
My bad, I read the left had side as brmp and right hand side as brms.